src/components/SimpleSelect.js

```diff
@@ -82,17 +74,29 @@ class SimpleSelect extends React.Component {
           {this.props.children ?
             this.props.children :
             (this.props.items.map((item, i) => {
+              const { icon, isSelected, onClick, text, ...rest } = item;
+
               return (
                 <Option
+                  isSelected={isSelected}
                   key={i}
-                  label={item.text}
-                  onClick={this._handleItemClick.bind(null, item)}
+                  label={text}
+                  onClick={e => {
+                    if (this.props.scrimClickOnSelect) {
+                      this.props.onScrimClick(e);
+                    }
+
+                    if (onClick && typeof onClick === 'function') {
```

Comment:
We could set a defaultProp instead of the if?

Reply:
good call.  I'll change it.

Reply:
yeah good call

Reply:
oh, no I can't because the onClick is a property in the item object in the items array prop